cloud/pkg/controller/controller/upstream.go

```diff
@@ -331,6 +331,24 @@ func (uc *UpstreamController) updateNodeStatus(stop chan struct{}) {
 					log.LOGGER.Warnf("message: %s process failure, update node failed with error: %s, namespace: %s, name: %s", msg.GetID(), err, getNode.Namespace, getNode.Name)
 					continue
 				}
+
+				resMsg := model.NewMessage(msg.GetID())
+				resMsg.Content = "update node status successfully"
```

Comment:
nit: in other places in KubeEdge, we send "OK" as response content. Lets keep it consistent